__tests__/postcodeInfoClient.ts

```diff
@@ -0,0 +1,22 @@
+import { PostcodeInfoClient } from '../src'
+
+const postcodeInfoClient: PostcodeInfoClient = new PostcodeInfoClient('<gibberish token>')
+
+describe('postcodeInfoClient', () => {
```

Comment:
Tests are WIP currently, will either mock, or set it up to run against the real api / both.